django/db/migrations/operations/models.py

```diff
@@ -802,6 +805,7 @@ def state_forwards(self, app_label, state):
         model_state = state.models[app_label, self.model_name_lower]
         indexes = model_state.options[self.option_name]
         model_state.options[self.option_name] = [idx for idx in indexes if idx.name != self.name]
+        state.reload_model(app_label, self.model_name_lower, delay=True)
```

Comment:
My impression is that tests that need database tables go in `test_operations.py`, other tests go in `test_state.py`. I don't think these tests interact with the db, do they?